rmgpy/data/kinetics/family.py

```diff
@@ -1217,7 +1217,7 @@ def applyRecipe(self, reactantStructures, forward=True, unique=True):
                     for i in range(4,highest+1):
                         atomLabels['*{0:d}'.format(i)].label = '*{0:d}'.format(4+highest-i)
                         
-            elif label == 'Intra_ene_reaction':
+            elif label == 'intra_ene_reaction':
```

Comment:
Please fix this in the original commit

Reply:
Will do! Must have squashed into the wrong commit.